jenkins/ci.fedora.Dockerfile

```diff
@@ -71,7 +71,27 @@ RUN \
         `# Base command used by CI unit test jobs` \
         `#` \
         tox \
-    && \
+        `#` \
+        `# Required for running various linters` \
+        `#` \
+        black \
+        npm \
+        python3-flake8 \
+        `#` \
+        `# Required for building RPMs` \
+        `#` \
+        python3-jinja2-cli \
+        rpmlint \
+        http://hdn.corp.redhat.com/rhel8-csb/RPMS/noarch/redhat-internal-cert-install-0.1-25.el7.noarch.rpm \
```

Comment:
Why is this in the public repo?

Reply:
Well...I guess that's a good question.

The logic goes like, "The CI container needs those certs, so they are installed as part of the container recipe, which we currently store in the public repo...."  But, yes, that's an internal network name, reachable only through the tunnel.

So, how should we proceed?